### PR TITLE
Multiple Servers

### PR DESCRIPTION
**Pull request by Ivans51**
Signed-off-by: Ivan Del Pino <idelpino@teclib.com>

### Changes description

Added Button open Activity list servers
Added Activity list servers
Added Layout list servers
Added Dialog
screenshot_1539640334
screenshot_1539640415
screenshot_1539640418
screenshot_1539640520
screenshot_1539640530
screenshot_1539640543

### Checklist

Please check if your PR fulfills the following specifications:

- [x] Tests for the changes have been added
- [ ] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@ivans51|16|
|@rafaelje|2|

### References

Closes: https://github.com/flyve-mdm/android-inventory-agent/issues/90
Related #N/A
Depends on #N/A

### CODE REVIEW

File: app/src/main/java/org/flyve/inventory/agent/ui/ActivityListServers.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * Flyve MDM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Rafael Hernandez
```
Comment (ajsb85) [inline app/src/main/java/org/flyve/inventory/agent/ui/ActivityListServers.java]: If you @Ivans51 are creating this file, why you added Rafael as author?

Comment (Ivans51) [reply app/src/main/java/org/flyve/inventory/agent/ui/ActivityListServers.java]: Ok, I'm going to change it. I didn't see it

**Comment by ajsb85** [comment]
Hi, @ticgal
Your feature request.
Close to be ready.

**Comment by rafaelje** [comment]
need to fix conflict

**Pull request merged**